### ISSUE TITLE
System.import

### ISSUE THREAD

**Issue by danielfalk** [normalize/discussions#37]
Normalize.io looks so promising, however it looks like it is designed to leave out loading modules on demand.  This is unfortunate, as large apps don't always want to load all assets up front.  The System.import() api would work well for this.  Would you consider supporting it?

**Comment by jonathanong** [comment]
Yes but you'll need a custom module loader that normalize may or may not provide
